database/factories/Hold/HoldFactory.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Database\Factories\Hold;
+
+use App\Models\Hold\Hold;
+use App\Models\Navigation\Navaid;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class HoldFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = Hold::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+            'navaid_id' => Navaid::factory()->create()->id,
+            'inbound_heading' => $this->faker->numberBetween(0, 360),
+            'minimum_altitude' => $this->faker->numberBetween(7000, 24000),
+            'maximum_altitude' => $this->faker->numberBetween(7000, 24000),
```

Comment:
Strictly speaking, needs to be greater than minimum, if that's possible with Faker

Reply:
I'm not sure it is sadly, I had a quick look into it. I guess if you're testing something specific, you'd override the attributes when generating the model anyway

Reply:
Good point!